package for_errors

func main() {
    for i := 1; true; i/0 {
        
    }
}
